source/code/source/playfab/QoS/PlayFabQoSApi.cpp

```diff
@@ -41,7 +41,7 @@ namespace PlayFab
             }
         }
 
-        void OnWriteEventsResult(int httpCode, std::string result, std::shared_ptr<CallRequestContainerBase> reqContainer)
+        void OnWriteEventsResult(int, std::string result, std::unique_ptr<CallRequestContainerBase> reqContainer)
```

Comment:
We changed it to shared_ptr, so this will need to be a shared_ptr

Reply:
I didn't see your name on the git blame, so I thought it wasn't that particular change. Thanks for pointing it out